cmake/kokkos_arch.cmake

```diff
@@ -615,7 +622,7 @@ IF (KOKKOS_ENABLE_OPENMPTARGET)
     COMPILER_SPECIFIC_FLAGS(
       Clang -Xopenmp-target -march=${CLANG_CUDA_ARCH} -fopenmp-targets=nvptx64
       XL    -qtgtarch=${KOKKOS_CUDA_ARCH_FLAG}
-      NVHPC -gpu=${NVHPC_CUDA_ARCH}
+      NVHPC -Minfo=mp -mp=gpu -gpu=${NVHPC_CUDA_ARCH}
```

Comment:
The `NVHPC -Minfo=mp -mp=gpu -gpu=${NVHPC_CUDA_ARCH}` option is to compile the OpenMPTarget backend using NVHPC compiler; this is just for testing purpose (comparing the OpenACC backend and OpenMPTarget backend), not needed for the OpenACC backend. 
We can delete this line since NVHPC compiler is not an official compiler supporting the OpenMPTarget backend.

Reply:
I think in this pr we should not add anything for building other backends with NVHPC. And if you drop the flags we could see if the CI failure in the serial backend is the reason for the failure

Reply:
`-Minfo=mp` is optional, but `-mp=gpu` is necessary for correct compilation enabling target offloading. Maybe, we can add `-Minfo=mp` option only in the debugging mode.